src/Raven.Client/Documents/LinqExtensions.cs

```diff
@@ -875,6 +889,20 @@ public static IOrderedQueryable<T> OrderByScoreDescending<T>(this IQueryable<T>
             return (IOrderedQueryable<T>)queryable;
         }
 
+        /// <summary>
+        /// Perform an initial sort by lucene score descending.
+        /// </summary>
+        public static IOrderedQueryable<T> ThenByScoreDescending<T>(this IQueryable<T> self)
```

Comment:
We need to change this to IOrderedQueryable IIRC

Reply:
fixed